### PR TITLE
Resolved missing packages and project related files while performing commit.

### PR DESCRIPTION
**Pull request by daisukenishino**
https://github.com/SymphonyTeleca/SampleProgram/pull/10

### CODE REVIEW

File: ASPNET/SignalR/SignalRChat/ChatWindow.aspx [hunk 1]
```diff
@@ -27,6 +29,7 @@
 
         function SendClick() {
             if (event.keyCode == 13) {
+                window.event.returnValue = false;
```
Comment (yusukemaegawa) [inline ASPNET/SignalR/SignalRChat/ChatWindow.aspx]: @daisukenishino 

I think that this program works only in IE.
Do you plan to support cross browser?

**Comment by yusukemaegawa** [comment]
@daisukenishino 

>https://github.com/SymphonyTeleca/SampleProgram/pull/10

Above article describes as follows.

>@daisukenishino 
>We found javascript issues while testing in internet explorer 9.
>We fixed these issues and did code commit.

Microsoft does not support IE9.
Does this issue happen in IE11?
If this issue does not happen in IE11, is it necessary to modify this issue?

**Comment by daisukenishino** [comment]
@yusukemaegawa 
I think that there is a needs to verify operation in IE11 too.

@supragyan 
Please check whether original code cause problems with IE11,
and If the problem will be occured in IE11 too, I will merge this fix.
However, there is a needs to check that the modified code to work correctly even IE11.

**Comment by daisukenishino** [comment]
@kishoreandukuri

Can you perform this check in @supragyan surrogate ?

**Comment by kishoreandukuri** [comment]
@daisukenishino 
>Can you perform this check in @supragyan surrogate ?
Sure, I will perform

**Comment by daisukenishino** [comment]
@kishoreandukuri 

Please tell us about this status.
If this pullrequest not be merged, We can not merge the subsequent pullrequest.

**Comment by kishoreandukuri** [comment]
@daisukenishino 
>Please tell us about this status.
It is working in IE11 as expected.

>If this pullrequest not be merged, We can not merge the subsequent pullrequest.
please merge the pull request.

**Comment by daisukenishino** [comment]
@yusukemaegawa 

>https://github.com/OpenTouryoProject/SampleProgram/pull/11#issuecomment-179700916

please merge the pull request.

**Comment by yusukemaegawa** [comment]
@daisukenishino 

>[#11 (comment)](https://github.com/OpenTouryoProject/SampleProgram/pull/11#discussion_r50649029)

Please reply to above comment.

**Comment by daisukenishino** [comment]
@yusukemaegawa 

> https://github.com/OpenTouryoProject/SampleProgram/pull/11#issuecomment-180137932

Because is sample program, currently it support only IE11.
In the future, we will consider to support of the modern browser.

please merge the pull request.

**Comment by yusukemaegawa** [comment]
@daisukenishino 

>Because is sample program, currently it support only IE11.
>In the future, we will consider to support of the modern browser.

I see.

**Pull request merged**